src/zepto.js

```diff
@@ -286,7 +286,9 @@ var Zepto = (function() {
   }
 
   function setAttribute(node, name, value) {
-    value == null ? node.removeAttribute(name) : node.setAttribute(name, value)
+    node.setAttribute ?
+      (value == null ? node.removeAttribute(name) : node.setAttribute(name, value))
+      :node.prop(name, value)
```

Comment:
What is the `node.prop()` method? As far as I can tell, `node` here is a plain DOM node. There is no `prop()` method in the DOM.